src/biokbase/narrative/tests/conftest.py

```diff
@@ -0,0 +1,15 @@
+"""General configurations and fixtures for the narrative tests."""
+
+import vcr
```

Comment:
use the excellent `vcrpy` package to record and replay API responses. The following YAML files are all recorded responses of the workspace queries made whilst testing `putative-ref` and its sibling transform types. They can be regenerated easily by removing the .yaml files and ensuring that there is a valid token for the test_user identity.